iroha_actor/src/broker.rs

```diff
@@ -64,95 +55,19 @@
 use std::any::{Any, TypeId};
 use std::{collections::HashMap, sync::Arc};
 
-use dashmap::DashMap;
-use futures::stream::{FuturesUnordered, StreamExt};
+use dashmap::{mapref::entry::Entry, DashMap};
+use futures::future;
 
 use super::*;
 
-const CHANNEL_SUBSCRIBER_SIZE: usize = 100;
-
-/// Type alias for Message Type.
-pub type MessageType = TypeId;
-
-/// Type alias for Actor Type.
-pub type ActorType = TypeId;
-
-/// Type alias for Subscription Id.
-pub type SubscriptionId = u128;
-
-/// Channel to the Actor.
+type TypeMap<V> = DashMap<TypeId, V>;
```

Comment:
Type map is not a good name, I believe this is what has lead us to the bug here previously. In the previous PR I added aliases and moved things into subscribers struct exactly for this - to improve readability and understandability. We need to know that first key is message type id, then actor id, and already then recipient.